lib/node_modules/@stdlib/lapack/base/dlacpy/src/dlacpy.c

```diff
@@ -0,0 +1,212 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/lapack/base/dlacpy.h"
+#include "stdlib/blas/base/shared.h"
+#include "stdlib/strided/base/stride2offset.h"
+#include "stdlib/math/base/special/fast/min.h"
+#include "stdlib/ndarray/base/assert/is_row_major.h"
+
+/**
+* Copies all of a matrix `A` to another matrix `B`.
+*
+* @param M         number of rows in matrix `A`
+* @param N         number of columns in matrix `A`
+* @param A         input matrix
+* @param strideA1  stride of the first dimension of `A`
+* @param strideA2  stride of the second dimension of `A`
+* @param offsetA   starting index for `A`
+* @param B         output matrix
+* @param strideB1  stride of the first dimension of `B`
+* @param strideB2  stride of the second dimension of `B`
+* @param offsetB   starting index for `B`
+*/
+void API_SUFFIX(c_copy_all)( const CBLAS_INT M, const CBLAS_INT N, const double *A, const CBLAS_INT strideA1, const CBLAS_INT strideA2, const CBLAS_INT offsetA, double *B, const CBLAS_INT strideB1, const CBLAS_INT strideB2, const CBLAS_INT offsetB ) {
```

Comment:
These should be private functions. Right now, you are exposing them outside the translation unit. That is not desired.